helm/templates/configmap.yaml

```diff
@@ -12,7 +12,7 @@ data:
       {{ $dst }} {
         {{- range $key, $value := (index .Values "configMap" (printf "%s" $dst)) }}
         {{- if $value }}
-        {{ $key }} = {{ $value }}
+        {{ $key }} = {{ $value | quote }}
```

Comment:
This will quote ports too right? Is that a problem?

Reply:
I also had the same concern (https://github.com/lightbend/config/blob/master/HOCON.md#automatic-type-conversions). So, we tested by making changes locally.

Reply:
Gotcha - saw port missing in the PR description, and thought it might have been omitted in the verification :)